ci-operator/config/openshift/verification-tests/openshift-verification-tests-release-4.15__ota-multi-stable-4.15-upgrade-from-stable-4.14.yaml

```diff
@@ -0,0 +1,86 @@
+base_images:
+  ansible:
+    name: "4.14"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    prerelease:
+      architecture: multi
+      product: ocp
+      version_bounds:
+        lower: 4.14.0-0
+        upper: 4.15.0-0
+  target:
+    prerelease:
+      architecture: multi
+      product: ocp
+      version_bounds:
+        lower: 4.15.0-0
+        upper: 4.16.0-0
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: azure-ipi-amd-mixarch-f28
+  cron: 54 6 14,28 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      ADDITIONAL_WORKER_VM_TYPE: Standard_D4ps_v5
+      ADDITIONAL_WORKERS: "1"
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "2"
+      ENABLE_OTA_TEST: OCP-53907
+    test:
+    - ref: ipi-install-heterogeneous
+    - ref: cucushift-upgrade-healthcheck
+    - ref: cucushift-ota-preupgrade
+    - ref: cucushift-upgrade-tolatest
+    - ref: cucushift-ota-postupgrade
+    - chain: cucushift-installer-check-cluster-health
+    workflow: cucushift-installer-rehearse-azure-ipi
+zz_generated_metadata:
+  branch: release-4.15
```

Comment:
To keep consistent with installation dedicated ci, I suggest to use `master` branch here, we already indicated release version in `variant`.

Reply:
sure